### DISCUSSION TITLE
Which score to use for differential motif identification

### DISCUSSION THREAD

**Discussion by Jia340** [stuart-lab/signac#1217] [Q&A]
Hi,

Thanks for the amazing package for multiome data analysis! I have a question regarding using FindMarkers to identify differential motifs between two cell types. In chromVar's original paper, they used the corrected deviation score instead of z-scores for differential test (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623146/, Fig2B and Methods). It appears that their z-score is not a metric normalized across samples/cells, but just to take the deviation of background into consideration. So using the corrected deviation score directly makes sense to me. 

However, in signac RunChromVar, it seems that only the z-score matrix is returned and the FindMarker() function is run based on z-scores. Could anyone help me understand the reason behind this? Thank you!

**Comment by RaghadShu** [comment]
I have the same question.. But I believe the z-score is the same as the deviation score (from [ArchR](https://www.archrproject.com/bookdown/chromvar-deviatons-enrichment-with-archr.html) documentation). No?
